src/PrestaShopBundle/Install/Install.php

```diff
@@ -293,18 +324,89 @@ public function clearDatabase($truncate = false)
     {
         $instance = Db::getInstance();
         $instance->execute('SET FOREIGN_KEY_CHECKS=0');
-        $sqlRequest = (($truncate) ? 'TRUNCATE' : 'DROP TABLE');
         foreach ($instance->executeS('SHOW TABLES') as $row) {
             $table = current($row);
-            if (!_DB_PREFIX_ || preg_match('#^'._DB_PREFIX_.'#i', $table)) {
-                $sqlRequest .= ' `'.$table.'`,';
+            if (!_DB_PREFIX_ || preg_match('#^' . _DB_PREFIX_ . '#i', $table)) {
+                $instance->execute((($truncate) ? 'TRUNCATE TABLE ' : 'DROP TABLE ').'`' . $table . '`');
             }
         }
-        $instance->execute(rtrim($sqlRequest, ','));
+
         $instance->execute('SET FOREIGN_KEY_CHECKS=1');
     }
 
     /**
+     * Install Cldr Datas
+     */
+    public function installCldrDatas()
+    {
+        $cldrUpdate = new Update(_PS_TRANSLATIONS_DIR_);
+        $cldrUpdate->init();
+
+        //get each defined languages and fetch cldr datas
+        $langs = \DbCore::getInstance()->executeS('SELECT * FROM '._DB_PREFIX_.'lang');
```

Comment:
Using classes with `Core` prevents all override to be used.

Please use `Db` instead, and declare it in the list of `use [...]` statements.

Reply:
I'm not sure we want to call override in case of an install?

Reply:
This may not happen, but this issue made us work on the whole src/ folder to remove these `<class>Core` calls